airflow/bin/cli.py

```diff
@@ -336,6 +336,47 @@ def list_tasks(args, dag=None):
         print("\n".join(sorted(tasks)))
 
 
+def parsedate_enhanced(args, dag):
+    '''
+    In addition to supporting date parsing, this also supports
+    @now,
+    @start_date (i.e. dag start date),
+    @end_date (i.e. dag end date if one exists), and
+    @latest (i.e. run closest to and before now that honors the schedule interval)
+    '''
+    timestr = args.execution_date
+    now = datetime.now()
+    start_date = dag.default_args['start_date'] if \
```

Comment:
Thought #2 -- there are actually a few different "start dates"
1. `dag.start_date` (could be `None`)
2. `dag.default_args['start_date']` (could be `None` or not even exist)
3. `dag.tasks[i].start_date` (could be `None`)

In the schedulers logic, if a DAG has a start date, that takes precedence. If it doesn't, then `min(t.start_date for t in dag.tasks)` is checked. Each task's start_date is either explicitly provided OR taken from default_args. 

I'm not sure if that level of detail is required, but it occurs to me that `default_args` might not be a robust place to look for the start date